csclient/params/helpers.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package params
+
+import (
+	"gopkg.in/juju/charm.v6-unstable/resource"
+)
+
+// Resource2API converts a charm resource into an API Resource struct.
+func Resource2API(res resource.Resource) Resource {
```

Comment:
I take it back... given that we need a helper for both sides of the API... I guess it has to be here.  Kinda wish it could live somewhere else and just import params... but I don't think it's worth creating a new package at this time.